services/project-grafana-loki/0.48.5/defaults/cm.yaml

```diff
@@ -6,6 +6,23 @@ metadata:
   namespace: ${releaseNamespace}
 data:
   values.yaml: |
+    ####################################################################
+    ## BEGIN DKP specific config overrides                            ##
+    ## This is added as a workaround to use the same configmap for    ##
+    ## both project-grafana-loki & object-bucket-claims helmreleases. ##
+    ####################################################################
+    dkp:
+      project-grafana-loki:
+        enabled: true
+        enableOBCHealthCheck: false
+        bucketName: proj-loki-${releaseNamespace}
+        storageClassName: dkp-object-store
+        additionalConfig:
+          maxSize: "20G"
```

Comment:
at the workspace level we default to 50G, do we want these to be the same?

Reply:
I intentionally set it lower than WS. I guess for realworld workload customer needs to override it anyways.